internal/cmd/commands/scopescmd/funcs.go

```diff
@@ -17,30 +23,38 @@ func init() {
 
 func extraActionsFlagsMapFuncImpl() map[string][]string {
 	return map[string][]string{
-		"create": {"skip-admin-role-creation", "skip-default-role-creation"},
+		"create": {flagSkipAdminRoleCreationName, flagSkipDefaultRoleCreationName},
+		"update": {flagPrimaryAuthMethodIdName},
 	}
 }
 
 type extraCmdVars struct {
 	flagSkipAdminRoleCreation   bool
 	flagSkipDefaultRoleCreation bool
+	flagPrimaryAuthMethodId     string
 }
 
 func extraFlagsFuncImpl(c *Command, set *base.FlagSets, f *base.FlagSet) {
 	for _, name := range flagsMap[c.Func] {
 		switch name {
-		case "skip-admin-role-creation":
+		case flagSkipAdminRoleCreationName:
 			f.BoolVar(&base.BoolVar{
-				Name:   "skip-admin-role-creation",
+				Name:   flagSkipAdminRoleCreationName,
 				Target: &c.flagSkipAdminRoleCreation,
 				Usage:  "If set, a role granting the current user access to administer the newly-created scope will not automatically be created",
 			})
-		case "skip-default-role-creation":
+		case flagSkipDefaultRoleCreationName:
 			f.BoolVar(&base.BoolVar{
-				Name:   "skip-default-role-creation",
+				Name:   flagSkipDefaultRoleCreationName,
 				Target: &c.flagSkipDefaultRoleCreation,
 				Usage:  "If set, a role granting the anonymous user access to log into auth methods and a few other actions within the newly-created scope will not automatically be created",
 			})
+		case flagPrimaryAuthMethodIdName:
+			f.StringVar(&base.StringVar{
+				Name:   flagPrimaryAuthMethodIdName,
+				Target: &c.flagPrimaryAuthMethodId,
+				Usage:  "If set, the primary auth method id for the scope.  A primary auth method is allowed to autovivify users and is used as a source for account full name and email for a scope's users",
```

Comment:
I think most people won't know what autovivify means. I would change it to "create users on first login" or so.

Reply:
ty!